### PR TITLE
add first patch of cime tests to public CI

### PR DESCRIPTION
**Pull request by mahf708**
Adds the first batch of 7 tests to run on public CI resources during PR submission.

Tests are SMS, SMS_D, ERS, ERP, REP, PET, PEM using F2010 and ne4pg2_oQU480. Tests are run with _P4 because CI resources are limited to 4 virtual CPUs for now.

[BFB]

### CODE REVIEW

**Review by jgfouca — APPROVED** [review]
Nice!

File: .github/workflows/e3sm-gh-ci-cime-tests.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          - "REP"   # 0h30m
+          - "ERS"   # 1h00m
+          - "ERP"   # 1h15m
+          - "ERR"   # 1h00m
+          - "PET"   # 0h35m
+          - "PEM"   # 0h45m
+          - "SEQ"   # 3h10m
+          - "ERI"   # Exception from case_test: [Errno 2] No such file or directory:
+          - "ERT"   # Exception from case_test: ERROR: No variable AVG_HIST_N found in case
+          - "PEA"   # ERROR: BUILD FAIL: build e3sm failed
```
Comment (rljacob) [inline .github/workflows/e3sm-gh-ci-cime-tests.yml:29]: please remove theses tests:  ERT (a longer ERS),  ERR (ERS with short-term acrchiving), ERI (hybrid run).  Also PEA (compares 1 task MPI with mpi-serial).  Also remove the SEQ test (have no idea why that's 3 hours but also needs a staggered layout for a fully coupled case to work).

**Comment by rljacob** [comment]
Did you try a fully coupled case on these CI resources?

For artifacts to save, please add files ending in cprnc.out

**Comment by rljacob** [comment]
I downloaded one of the archive files and most of it is what we want.  We don't need anything from bld/cmake-bld or the env_archive_test directory.

**Comment by rljacob** [comment]
One more thing we need in the archive:  TestStatus.log which is the casedir.

**Comment by jgfouca** [comment]
Yeah, the TestStatus.log is important. Also, I wonder if it would be easier to add an entry for singularity machine in config_pes.xml to just set the procs to 4 for all grids/compsets? That way you don't have to add `_P4` to all the tests.

**Comment by rljacob** [comment]
Most of those passing integration tests are at resolutions that are going to change so we should wait until https://github.com/E3SM-Project/E3SM/pull/6202 is merged.  Then update the CI tests.  Just go with what you have.

**Comment by rljacob** [comment]
@mahf708 I cleaned up the PR description (above the line).   You may want to edit your first-person story (below the line).

**Review by rljacob — APPROVED** [review]

**Pull request merged**